docker-storage-setup.sh

```diff
@@ -823,6 +830,13 @@ setup_storage() {
   fi
 }
 
+reset_storage() {
+    if [ "$STORAGE_DRIVER" == "devicemapper" ]; then
+	reset_lvm_thin_pool
+    fi
+    rm -f $DOCKER_STORAGE
```

Comment:
Seems like it'd be a bit friendlier to reset this to the default.  On an Atomic Host system, we could actually just do:

```
cp /{usr/,}etc/sysconfig/docker-storage
```

On a yum-managed system this isn't available though, so we could just test for it, like:

```
if test -f /usr/$DOCKER_STORAGE; then
  cp /usr/$DOCKER_STORAGE $DOCKER_STORAGE
else
  rm -f $DOCKER_STORAGE
fi
```
?